### ISSUE TITLE
Setting length in get object results in data from the end of object instead of front

### ISSUE THREAD

**Issue by fkautz** [minio/minio-py#186]
Getting an object with a length limit returns data from the end of the object instead of the beginning
```py
# get an object with range
result = client.get_object(bucket, 'string_object', length=5)
for c in result:
    print(c)
```

**Comment by fkautz** [comment]
object contents: 'hello world'

```py
# Correct behavior
>>> result = client.get_object(bucket, 'string_object', offset=0, length=5)
>>> for c in result:
...     print(c)
... 
hello

# Incorrect behavior, expecting same result as first example
>>> result = client.get_object(bucket, 'string_object', length=5)
>>> for c in result:
...     print(c)
... 
orld
```

**Comment by fkautz** [comment]
Setting length only should prepend 0

incorrect:
```
bytes=-4
```

correct:
```
bytes=0-4
```

**Comment by fkautz** [comment]
Fixed with #187.
